content/fr/observability.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Observability
+status: Feedback Appreciated
+category: concept
+tags: ["property", "", ""]
+---
+
+L'observabilité est une propriété des systèmes qui définit le degré auquel le système peut générer des informations exploitables.
+Elle permet aux utilisateurs de comprendre l'état d'un système à partir de ses sorties externes et de prendre des mesures (correctives).
+
+Les systèmes informatiques sont mesurés en observant des signaux de bas niveau tels que le temps CPU, la mémoire, l'espace disque, et des signaux de niveau supérieur et métiers, notamment les temps de réponse de l'API, les erreurs, le nombre de transactions par seconde, etc.
```

Comment:
A mon sens tu dois avoir une concordance entre "niveau supérieur" et "métiers". J'aurai tendance à mettre du pluriel

Reply:
Ici, ce sont les "signaux métiers". 
Le passage en anglais est "low-level signals such as CPU time, memory, disk space, and higher-level and business signals".

Reply:
Il est possible aussi de répéter "signaux", "des signaux de niveau supérieur et des signaux métiers" ou mettre une virgule avant "des signaux de niveau supérieur, et métiers".

Reply:
Mettre "des signaux de niveau supérieur et des signaux métiers"

Reply:
Modification effectuée